### DISCUSSION TITLE
[Suggestion] file-manager Enlarge media preview thumbnail

### DISCUSSION THREAD

**Discussion by raspin0** [linuxdeepin/developer-center#3145] [Features Request & Ideas | 特性请求 & 头脑风暴]
Several people already have reported that media file preview on deepin is surprisingly small. I wonder if making them bigger would be viable on deepin.

Ubuntu:
![image](https://user-images.githubusercontent.com/39202451/167284445-c9cade56-a1a2-4ecc-a80b-c6a5712d6c5e.png)
Deepin: 
![image](https://user-images.githubusercontent.com/39202451/167284518-f6e6311a-2e7b-4145-b567-df0e99584379.png)

**Comment by BLumia** [comment]
![image](https://user-images.githubusercontent.com/10095765/167356859-d5c9cbad-2103-4b00-8c9b-5f4b07c53ec7.png)

It can be adjusted with this slider :D

**Reply by raspin0** [reply]
It's not about resizing the entire item, but thumbnail inside.
![Bez nazwy](https://user-images.githubusercontent.com/39202451/167371366-a3408874-dd44-424d-9bd5-c6ebcb697541.png)

**Reply by BLumia** [reply]
This is probably a design choice, @meiyixiang could you take a look? Thanks!

**Comment by meiyixiang** [comment]
As far as I know, it is because the research and development failed to achieve the effect of UI design. Now we are making a brand new desktop design, and we will consider this problem together in the next generation
